frontend/lib/src/components/widgets/DataFrame/columns/ProgressColumn.test.ts

```diff
@@ -159,6 +159,7 @@ describe("ProgressColumn", () => {
     [0.12, "percent", "12%"],
     [1100, "compact", "1.1K"],
     [-1234.567, "accounting", "(1,234.57)"],
+    [1000000, "bytes", "1MB"],
```

Comment:
Can you add a 2-3 more cases to check that its correctly formatting into different units.

Reply:
Done.  One thing to note is that I found an issue with the Intt.Numberformat in that ti reports `BB` instead of `GB`.